MentoringHomeWork/HW9/AirportSimulation/Passport.cs

```diff
@@ -0,0 +1,24 @@
+using System.Net.Sockets;
+
+namespace AirportSimulation;
+
+public class Passport
+{
+    public string PassportNumber { get; set; }
+    public DateTime PassportDate { get; set; }
+    public DateTime PassportExpirationDate { get; }
+
+    public Passport(string passportNumber, DateTime passportDate, DateTime passportExpirationDate)
+    {
+        PassportNumber = passportNumber;
+        PassportDate = passportDate;
+        PassportExpirationDate = passportExpirationDate;
+
+        if (PassportExpirationDate <= DateTime.Today)
+        {
+            Console.WriteLine("Error: Passport has expired!");
+        }
+    }
+
+    public bool PassportIsValid() => !string.IsNullOrEmpty(PassportNumber) && PassportExpirationDate > DateTime.Today;
```

Comment:
The naming convention for boolean operations - IsSomethingTrueOfFalse
```suggestion
    public bool IsPasspordValid() => !string.IsNullOrEmpty(PassportNumber) && PassportExpirationDate > DateTime.Today;
```

And the question - do you need a method for this or just a prop? 

```suggestion
   public bool IsValid => !string.IsNullOrEmpty(PassportNumber) && PassportExpirationDate > DateTime.Today;
```

Reply:
resolved